apps/frontend/components/Report/FloodReport/FloodReport.tsx

```diff
@@ -23,9 +23,13 @@ export const FloodReport = ({
   isAllColumnReport: boolean;
 }) => {
   const report = useMemo(() => {
+    // eslint-disable-next-line no-console
+    console.log({ forms });
     const formattedForms = filterFloodReports(
       forms.map(form => formatFloodFields(form)),
     );
+    // eslint-disable-next-line no-console
+    console.log({ filteredForms: formattedForms });
```

Comment:
@wadhwamatic this temporarily, lets you see in the logs the difference between the fetched forms and the filtered ones

Reply:
Thanks @ericboucher but when I tested on the surge deployment, I'm not seeing any messages regarding forms in the console. Do I need to test locally?